docs/_user_guide/slurm.md

```diff
@@ -338,30 +330,29 @@ python3 investment-npv-serial.py
 
 Then submit the script:
 
-```bash
-$ sbatch investment-serial.slurm
+```bash title="Terminal Command"
+sbatch investment-serial.slurm
 ```
 
 You should see a similar output:
 
-```bash
+```{.yaml .no-copy title="Terminal Output"}
 Submitted batch job 44097
 ```
 
 Monitor your job:
-```bash
-$ squeue
+```bash title="Terminal Command"
+squeue
 ```
 
-The script should take less than 5 minutes to complete. Look at the slurm emails after the job is finished.
-Look at the output file.
+The script should take less than 5 minutes to complete. Look at the slurm emails after the job is finished. You can also review the output file.
 
-## Using `venv` Environment in Slurm Scripts
-We can also use a `venv` environment python instead of a system `python3` when running scripts via Slurm.
+### Using Virtual Environment in Slurm Scripts
+We can also employ a virtual Python enviuronment using `venv` instead of the system's `python3` when running scripts via Slurm.
 
-We can modify the slurm script to use previously created `venv` environment as follows:
+For example, let's modify the Slurm script to use a `venv` environment that we've already created located at `/zfs/gsb/intermediate-yens/venv/`:
```

Comment:
I would change this to have requirements.txt file, and directions on how to make, and source this venv in slurm